ui/components/app/app-components.scss

```diff
@@ -1,20 +1,19 @@
 /** Please import your files in alphabetical order **/
 @import 'account-list-item/index';
```

Comment:
I reverted all of the changes except those that are Flask-specific. I don't anticipate any problems, because the components whose imports changed are unrelated.

Reply:
Ref: d49fbd2